lib/runtime/wazero/imports.go

```diff
@@ -832,6 +832,7 @@ func ext_trie_blake2_256_root_version_2(ctx context.Context, m api.Module, dataS
 }
 
 func ext_trie_blake2_256_ordered_root_version_1(ctx context.Context, m api.Module, dataSpan uint64) uint32 {
+	// Why is this hardcoded to 0??
```

Comment:
we hard code to 0 because the routine to get the `blake2_256` hash for the trie is the same but only the version changes, so we use version 0 as default for `ext_trie_blake2_256_ordered_root_version_1`, the runtime still using this host function for extrinsic root state hash